### PR TITLE
Add Community Profile Files

### PR DESCRIPTION
**Pull request by jbants**

### CODE REVIEW

**Review by cuttlefish — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```
Comment (cuttlefish) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: We should probably remove the website specific example with something a little more stactools specific

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```
Comment (cuttlefish) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]: Would a shell session dump be more appropriate?

**Review by cuttlefish — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (jbants) [reply .github/ISSUE_TEMPLATE/bug_report.md:18]: :+1:

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (jbants) [reply .github/ISSUE_TEMPLATE/bug_report.md:24]: Something like `scriptcs/lint >> lint_errors.txt`

Comment (jbants) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Something like scripts/lint >> lint_errors.txt

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (gadomski) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]:
```suggestion
**To reproduce**
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+> Ex.
+>
+> 1. Install stactools and stactools-package dataset
+> 2. Run `scripts/test`
+> 3. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots and Shell session dumps**
```
Comment (gadomski) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]:
```suggestion
**Screenshots and shell session dumps**
```

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 5]
```diff
@@ -0,0 +1,15 @@
+Before you submit a pull request, please fill in the following:
+
+**Related Issue(s):**
+Please add links (if any) to related github issues.
+
+**Description:**
+Please explain the changes you made here.
+
+**PR Checklist:**
```
Comment (gadomski) [inline .github/PULL_REQUEST_TEMPLATE.MD:9]:
```suggestion
**PR checklist:**
```

File: CONTRIBUTING.MD [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+## Deprecation
+
+We recommend following the [Numpy Enhancement Proposals (NEP) 29](https://numpy.org/neps/nep-0029-deprecation_policy.html)
+suggested deprecation policy for supported python versions.
+
+>When a project releases a new major or minor version, we recommend that they support at least all minor versions of Python introduced and released in the prior 42 months from the anticipated release date with a minimum of 2 minor versions of Python, and all minor versions of NumPy released in the prior 24 months from the anticipated release date with a minimum of 3 minor versions of NumPy.
+
+## Submitting Pull Requests
+
+Any proposed changes to a existing stactools-packages should be done as pull requests. Please make these
```
Comment (gadomski) [inline CONTRIBUTING.MD:16]:
```suggestion
Any proposed changes to an existing stactools-packages should be done as pull requests. Please make these
```

**Review by gadomski — CHANGES_REQUESTED** [review]
Some copy-editing stuff, but otherwise looks great. Love the project board, never seen that before, seems super useful.

**Comment by jbants** [comment]
Thanks for the Review @gadomski and @cuttlefish.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (cuttlefish) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Meh, we can just leave it with the screenshot. I can't think of a concise way of telling people to copy/paste a command and its output without it sounding condescending.

**Review by gadomski — APPROVED** [review]

**Review by cuttlefish — APPROVED** [review]

**Pull request merged**